web/beacon-app/src/features/onboarding/components/StepButtons.tsx

```diff
@@ -2,28 +2,36 @@ import { Trans } from '@lingui/macro';
 import { Button } from '@rotational/beacon-core';
 import React from 'react';
 
+import { useOrgStore } from '@/store';
+
+import { ONBOARDING_STEPS } from '../shared/constants';
 type StepButtonsProps = {
   isSubmitting?: boolean;
   isDisabled?: boolean;
 };
 const StepButtons = ({ isSubmitting, isDisabled }: StepButtonsProps) => {
+  const state = useOrgStore((state: any) => state) as any;
+  const { currentStep } = state.onboarding as any;
```

Comment:
Will making the `Next` button disabled until a user provides an input be handled in a separate story?

Reply:
It seems we have a story for each form step that should handle this. In addition, I will discuss with @eschmier whether it is necessary to disable it since we already have input validation.